tests/ui/associated-inherent-types/multiple-candidates-in-adt-field-2.rs

```diff
@@ -0,0 +1,29 @@
+#![feature(inherent_associated_types)]
+#![expect(incomplete_features)]
+
+// Test that when we have an unnormalized projection we don't normalize it
+// to determine which IAT to resolve to.
```

Comment:
difference between this test and tests/ui/associated-inherent-types/candidate-with-alias-2.rs?

Reply:
whether the alias is on the impl self ty or the IAT self ty :thinking: probably won't have a case where we mess up one of these but not the other but still feels fine to have two tests :3